contractsV5/BytesLib.sol

```diff
@@ -0,0 +1,506 @@
+/*
+ * @title Solidity Bytes Arrays Utils
+ * @author Gonçalo Sá <goncalo.sa@consensys.net>
+ *
+ * @dev Bytes tightly packed arrays utility library for ethereum contracts written in Solidity.
+ *      The library lets you concatenate, slice and type cast bytes arrays both in memory and storage.
+ */
+
+pragma solidity ^0.5.0;
```

Comment:
we use only solidity 0.5.11 at the moment.
We fix the version.
without the ^
since we always want the same compilation result in production.

Reply:
Fixed